libs/core/src/components/pds-popover/stories/pds-popover.docs.mdx

```diff
@@ -0,0 +1,107 @@
+import { Meta, Canvas, ArgTypes } from '@storybook/blocks';
+
+import * as stories from './pds-popover.stories.js';
+
+<Meta of={stories}
+/>
+
+# Popover
+
+Popovers are containers that float on top of existing content used to display additional content.
+
+## Properties
+
+<ArgTypes of={stories} />
+
+## Variants
+
+### Trigger
+Whether your trigger is a `pds-button` or a different component or HTML element, it will be placed in the default slot between the opening and closing tags.
+There can only be one trigger for the Popover.
+
+#### Button trigger
+
+<Canvas mdxSource={`
+<pds-popover placement="bottom-right" html-content="true">
+  <div slot="content">
+    <p><strong>This is a popover</strong></p>
+  </div>
+  <pds-button variant="secondary">Click</pds-button>
+</pds-popover>
+`}>
+  <pds-popover placement="bottom-start" html-content="true">
```

Comment:
Just a thought, but since we now display our own source code, I think we could add something like <div style={{height: "200px"}}> to the examples here to show the entire popover and it will not display it in the code example.